osmoutils/binary_search_test.go

```diff
@@ -130,7 +154,7 @@ func TestErrTolerance_Compare(t *testing.T) {
 }
 
 func TestErrToleranceNonzero_Compare(t *testing.T) {
-	// Nonzero error tolerance test
+	// Nonzero error tolerance test, tolerance is used in context of searching for a solution for stable swap over a cfmm function
 	NonZeroErrTolerance := ErrTolerance{AdditiveTolerance: sdk.NewInt(10), MultiplicativeTolerance: sdk.Dec{}}
```

Comment:
Are we testing multiplicative tolerance anywhere? I think that should be tested as well

Reply:
We should test:
- only multiplicative is non zero
- both multiplicative and additive are non-zero